### ISSUE TITLE
install gems into vendor/cache

### ISSUE THREAD

**Issue by fjenett** [motionbank/piecemaker2-app#5]

**Comment by mattes** [comment]
I think the problem was that the App isn't using a system-wide ruby installation.

**Comment by fjenett** [comment]
Nah, it finds the globally installed gems, actually this is rather an issue to put onto motionbank/piecemaker2-api list ...
